Resources/Prototypes/Corvax/Entities/Objects/Devices/producrion.yml

```
@@ -0,0 +1,11 @@
+- type: entity
+  id: ThrusterSyndicateMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: двигатель Синдиката
+  components:
+  - type: MachineBoard
+    prototype: ThrusterSecurity
+    requirements:
+      AdvancedCapacitorStockPart: 3
+    stackRequirements:
+      Steel: 5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)